app/views/prior_authority/shared/_application_nav.html.erb

```diff
@@ -41,6 +43,9 @@
   <% elsif application_summary.expired? %>
     <br>
     <strong><%= t('.date_expired') %></strong> <%= application_summary.date_updated_str %>
+  <% elsif application_summary.provider_updated? %>
+    <br>
+    <strong><%= t('.date_amended') %></strong> <%= application_summary.date_updated_str %>
```

Comment:
**nit (and not sure i would prefer it myself)** but wondering if this long `elsif` could be simplified to use the status to find the i18n key.


```erb
<% else %>
  <strong><%= t('.date_#{status}') %></strong> <%= application_summary.date_updated_str %>
<% end %>
```